client/flags/flags.go

```diff
@@ -157,3 +156,10 @@ func ParseGasSetting(gasStr string) (GasSetting, error) {
 		return GasSetting{false, gas}, nil
 	}
 }
+
+func markFlagRequired(cmd *cobra.Command, name string) {
+	err := cmd.MarkFlagRequired(name)
+	if err != nil {
+		panic(fmt.Sprintf("Can't make a %q falg required; err: %s", name, err))
```

Comment:
Error message should never be capitalized:

```suggestion
		panic(fmt.Sprintf("can't mark the flag %q required: %v", name, err))
```